Core/include/Acts/Propagator/DirectNavigator.hpp

```diff
@@ -72,6 +72,7 @@ class DirectNavigator {
       if (not r.initialized) {
         // Initialize the surface sequence
         state.navigation.navSurfaces = navSurfaces;
+        state.navigation.currentSurface = navSurfaces.front();
```

Comment:
Maybe we should check if we are on the first surface before doing this? I'm not sure the start position is always guaranteed to be on the first surface.

Reply:
In `Navigator.hpp` we do this during initialization https://github.com/acts-project/acts/blob/main/Core/include/Acts/Propagator/Navigator.hpp#L326

I think we should stick to this pattern here

Reply:
makes sense ... indeed the `initialize()` method for the `DirectNavigator` does nothing

Reply:
concerning check if we are on the first surface ... is there a way of retrieving that info from the state?